src/main/flight/failsafe.c

```diff
@@ -103,7 +103,16 @@ static const failsafeProcedureLogic_t failsafeProcedureLogic[] = {
                 FAILSAFE_CHANNEL_NEUTRAL,       // ROLL
                 FAILSAFE_CHANNEL_NEUTRAL,       // PITCH
                 FAILSAFE_CHANNEL_NEUTRAL,       // YAW
-                FAILSAFE_CHANNEL_HOLD           // THROTTLE
+#if !defined(USE_VARIABLE_PITCH)
+                FAILSAFE_CHANNEL_HOLD,          // THROTTLE
+#else
+    // woga65: cut throttle, hold collective (kind of auto gyro)
+                FAILSAFE_CHANNEL_NEUTRAL,       // THROTTLE (cut trottle regardless until sufficiently tested)
+                FAILSAFE_CHANNEL_HOLD,          // AUX1 (NOT USED / ALWAYS HOLD)
```

Comment:
Handling the collective channel mapping like the first 4 channels could skip having aux1 and aux2 here.

Reply:
> Handling the collective channel mapping like the first 4 channels could skip having aux1 and aux2 here.

This would be true if COLLECTIVE + GYRO_GAIN were CH5 + CH6. Then, CONTROL_CHANNEL_COUNT would be 6 and nothing to skip at all.

Reply:
I am thinking along the lines of CONTROL_CHANNEL_COUNT=5, and gyro gain being handled as any adjustment in inav. Need to look into the code to see how feasible it would be.

INAV already has a way to adjust yaw gains in flight.

Reply:
Had a quick look,but haven't really tested it.

But maybe something like this:

```--- a/src/main/rx/rx.c
+++ b/src/main/rx/rx.c
@@ -65,7 +65,7 @@
 #include "rx/mavlink.h"
 #include "rx/sim.h"
 
-const char rcChannelLetters[] = "AERT";
+const char rcChannelLetters[] = "AERTC";
 
 static uint16_t rssi = 0;                  // range: [0;1023]
 static timeUs_t lastMspRssiUpdateUs = 0;
@@ -527,7 +527,12 @@ void parseRcChannels(const char *input)
     for (const char *c = input; *c; c++) {
         const char *s = strchr(rcChannelLetters, *c);
         if (s && (s < rcChannelLetters + MAX_MAPPABLE_RX_INPUTS))
-            rxConfigMutable()->rcmap[s - rcChannelLetters] = c - input;
+            if(*s == 'C')
+            {
+                rxConfigMutable()->cyclic = c - input;
+            } else {
+                rxConfigMutable()->rcmap[s - rcChannelLetters] = c - input;
+            }
     }
 }
 
diff --git a/src/main/rx/rx.h b/src/main/rx/rx.h
index 4d5e2b135..881a22d91 100644
--- a/src/main/rx/rx.h
+++ b/src/main/rx/rx.h
@@ -107,6 +107,7 @@ PG_DECLARE_ARRAY(rxChannelRangeConfig_t, NON_AUX_CHANNEL_COUNT, rxChannelRangeCo
 typedef struct rxConfig_s {
     uint8_t receiverType;                   // RC receiver type (rxReceiverType_e enum)
     uint8_t rcmap[MAX_MAPPABLE_RX_INPUTS];  // mapping of radio channels to internal RPYTA+ order
+    uint8_t cyclic;
     uint8_t serialrx_provider;              // Type of UART-based receiver (rxSerialReceiverType_e enum). Only used if receiverType is RX_TYPE_SERIAL
     uint8_t serialrx_inverted;              // Flip the default inversion of the protocol - e.g. sbus (Futaba, FrSKY) is inverted if this is false, uninverted if it's true. Support for uninverted OpenLRS (and modified FrSKY) receivers.
     uint8_t halfDuplex;                     // allow rx to operate in half duplex mode. From tristate_e.
```

And when you are checking if the channel is in range, to activate a mode, you can always return false for the channel marked as cyclic, if you are in helicopter mode.